internal/k8s/controller.go

```diff
@@ -371,6 +371,7 @@ func NewLoadBalancerController(input NewLoadBalancerControllerInput) *LoadBalanc
 				Namespace: os.Getenv("POD_NAMESPACE"),
 				Name:      os.Getenv("POD_NAME"),
 			},
+			PolicyCount: lbc.getAllPolicies,
```

Comment:
nit: This is called PolicyCount but is actually an array of policies, could it be called Policies instead?